spec.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Semantic Specification
+---
+
+## Concepts and Terminology
+
+A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system.
+
+A **Span** represents a logical unit of work in the system that has a start time and an end time. Spans may be nested and ordered to model parent-child and casual relationships. A Trace can be thought of as a tree of Spans.
+
+Every Span has zero or more **Logs**, each of which being a timestamped message with an optional structured data payload of arbitrary size.
+
+Every Span may also have zero or more key/value **Tags**, which do not have timestamps and simply annotate the spans.
+
+A **Trace Context** encapsulates the smallest amount of state needed to describe a Span's identity within a larger, potentially distributed trace, sufficient to propagate the context of a particular trace between processes.
```

Comment:
Agree that it needs a bit more, although @slimsag - no, it's not responsible for parent-child relationships, not explicitly anyway. E.g. if you're familiar with Zipkin's (trace_id, span_id, parent_span_id) tuple, you may notice that  parent_span_id did not have to be propagated in-band across RPCs, it could've been reported off-band.

@bensigelman the current wording makes emphasis on TC being the _identity_ of the span, maybe we can rephrase  that it actually has two roles, describing the span identity, and a container for distributed context.